x-pack/metricbeat/module/istio/pilot/_meta/fields.yml

```diff
@@ -5,48 +5,57 @@
   release: beta
   fields:
     - name: xds.count
-      type: integer
+      type: long
       description: >
         Count of concurrent xDS client connections for Pilot.
     - name: xds.pushes
-      type: integer
+      type: long
       description: >
         Count of xDS messages sent, as well as errors building or sending xDS messages
         for lds, rds, cds and eds.
-    - name: xds.push.time.sec
+    - name: xds.push.time.ms.bucket.*
+      type: object
+      object_type: long
+      description: >
+        Total time in seconds Pilot takes to push lds, rds, cds and eds, histogram buckets in milliseconds.
+    - name: xds.push.time.ms.sum
       type: long
       description: >
-        Total time in seconds Pilot takes to push lds, rds, cds and eds.
+        Total time in seconds Pilot takes to push lds, rds, cds and eds, histogram buckets in milliseconds.
+    - name: xds.push.time.ms.count
+      type: long
+      description: >
+        Total time in seconds Pilot takes to push lds, rds, cds and eds, histogram buckets in milliseconds.
```

Comment:
The descriptions of the above three fields are the same. But I assume they should be slightly different, since these are actually different fields? Also, I think the `seconds` in the description is incorrect since the field has `ms` in it?

Reply:
👍